packages/fx-core/src/core/index.ts

```diff
@@ -408,6 +408,8 @@ class CoreImpl implements Core {
             supported = await this.isSupported();
             if (!supported) {
                 this.ctx.logProvider?.warning(`non Teams project:${workspace}`);
+            } else{
+                await this.readConfigs();
```

Comment:
I tried to read config by setting true here this.runWithErrorHandling<null>("open", **true**, ok(null), () => this.coreImpl.open(workspace)), but seems the root is not assigned before open() called. So I manually read config here.